src/arch/arm/common_arm.lds

```diff
@@ -64,10 +64,12 @@ SECTIONS
         *(.bss)
         *(COMMON) /* fallback in case '-fno-common' is not used */
 
-        /* 4k breakpoint stack */
+#if defined(CONFIG_ARCH_AARCH32) && !defined(CONFIG_ARM_HYPERVISOR_SUPPORT)
+        /* 4k breakpoint stack, needed for PMODE_ABORT on ARM32 */
         _breakpoint_stack_bottom = .;
```

Comment:
In order to improve the "documentaton" even more, the names could also change from `_breakpoint_stack_...` to "_abort_mode_stack_..." then, because that is what this really is. Unless the proof use the name, so changing it is not trivial. Breakpoints just happen to trap into abort mode.